UnitsNet/UnitsNet.WindowsRuntimeComponent-Net45.csproj

```diff
@@ -53,7 +53,186 @@ There is probably a better way to achieve this?
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs;" Exclude="obj\**;Properties\AssemblyInfo.cs" />
```

Comment:
This happens often, but please revert all changes to `.csproj` files. Only the wildcard include should remain.